libandroid-navigation-ui/src/main/res/layout-land/summary_peek_layout.xml

```diff
@@ -37,7 +37,7 @@
         app:layout_constraintTop_toTopOf="parent"
         tools:src="@drawable/ic_route_preview"/>
 
-    <ImageButton
+    <android.support.v7.widget.AppCompatImageButtonn
```

Comment:
There's a typo here *`AppCompatImageButtonn` note the extra last `n`, glad that our UI tests caught it 🎉 

```
android.view.InflateException: Binary XML file line #8: Binary XML file line #8: Error inflating class <unknown>
at android.view.LayoutInflater.inflate(LayoutInflater.java:539)
at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
at android.view.LayoutInflater.inflate(LayoutInflater.java:374)
at android.support.v7.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:469)
at android.support.v7.app.AppCompatActivity.setContentView(AppCompatActivity.java:140)
at com.mapbox.services.android.navigation.testapp.test.TestNavigationActivity.onCreate(TestNavigationActivity.java:28)
at android.app.Activity.performCreate(Activity.java:6237)
at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1107)
at android.support.test.runner.MonitoringInstrumentation.callActivityOnCreate(MonitoringInstrumentation.java:667)
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2369)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2476)
at android.app.ActivityThread.handleRelaunchActivity(ActivityThread.java:4077)
at android.app.ActivityThread.-wrap15(ActivityThread.java)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1350)
at android.os.Handler.dispatchMessage(Handler.java:102)
at android.os.Looper.loop(Looper.java:148)
at android.app.ActivityThread.main(ActivityThread.java:5417)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
Caused by: android.view.InflateException: Binary XML file line #8: Error inflating class <unknown>
at android.view.LayoutInflater.createView(LayoutInflater.java:645)
at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:764)
at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:704)
at android.view.LayoutInflater.rInflate(LayoutInflater.java:835)
at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798)
at android.view.LayoutInflater.inflate(LayoutInflater.java:515)
... 19 more
Caused by: java.lang.reflect.InvocationTargetException
at java.lang.reflect.Constructor.newInstance(Native Method)
at android.view.LayoutInflater.createView(LayoutInflater.java:619)
... 24 more
Caused by: android.view.InflateException: Binary XML file line #24: Binary XML file line #24: Error inflating class <unknown>
at android.view.LayoutInflater.inflate(LayoutInflater.java:539)
at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
at android.view.LayoutInflater.inflate(LayoutInflater.java:374)
at android.view.View.inflate(View.java:19784)
at com.mapbox.services.android.navigation.ui.v5.NavigationView.initializeView(NavigationView.java:511)
at com.mapbox.services.android.navigation.ui.v5.NavigationView.<init>(NavigationView.java:104)
at com.mapbox.services.android.navigation.ui.v5.NavigationView.<init>(NavigationView.java:98)
... 26 more
Caused by: android.view.InflateException: Binary XML file line #24: Error inflating class <unknown>
at android.view.LayoutInflater.createView(LayoutInflater.java:645)
at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:764)
at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:704)
at android.view.LayoutInflater.rInflate(LayoutInflater.java:835)
at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798)
at android.view.LayoutInflater.inflate(LayoutInflater.java:515)
... 32 more
Caused by: java.lang.reflect.InvocationTargetException
at java.lang.reflect.Constructor.newInstance(Native Method)
at android.view.LayoutInflater.createView(LayoutInflater.java:619)
... 37 more
Caused by: android.view.InflateException: Binary XML file line #8: Binary XML file line #40: Error inflating class android.support.v7.widget.AppCompatImageButtonn
at android.view.LayoutInflater.inflate(LayoutInflater.java:539)
at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
at android.view.LayoutInflater.inflate(LayoutInflater.java:374)
at android.view.View.inflate(View.java:19784)
at com.mapbox.services.android.navigation.ui.v5.summary.SummaryBottomSheet.initialize(SummaryBottomSheet.java:162)
at com.mapbox.services.android.navigation.ui.v5.summary.SummaryBottomSheet.<init>(SummaryBottomSheet.java:58)
at com.mapbox.services.android.navigation.ui.v5.summary.SummaryBottomSheet.<init>(SummaryBottomSheet.java:53)
... 39 more
Caused by: android.view.InflateException: Binary XML file line #40: Error inflating class android.support.v7.widget.AppCompatImageButtonn
at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:776)
at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:704)
at android.view.LayoutInflater.rInflate(LayoutInflater.java:835)
at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798)
at android.view.LayoutInflater.parseInclude(LayoutInflater.java:971)
at android.view.LayoutInflater.rInflate(LayoutInflater.java:831)
at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798)
at android.view.LayoutInflater.parseInclude(LayoutInflater.java:971)
at android.view.LayoutInflater.rInflate(LayoutInflater.java:831)
at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798)
at android.view.LayoutInflater.inflate(LayoutInflater.java:515)
... 45 more
Caused by: java.lang.ClassNotFoundException: Didn't find class "android.support.v7.widget.AppCompatImageButtonn" on path: DexPathList[[zip file "/system/framework/android.test.runner.jar", zip file "/data/app/com.mapbox.services.android.navigation.app.test-1/base.apk", zip file "/data/app/com.mapbox.services.android.navigation.app-1/base.apk"],nativeLibraryDirectories=[/data/app/com.mapbox.services.android.navigation.app.test-1/lib/arm, /data/app/com.mapbox.services.android.navigation.app-1/lib/arm, /data/app/com.mapbox.services.android.navigation.app.test-1/base.apk!/lib/armeabi-v7a, /data/app/com.mapbox.services.android.navigation.app-1/base.apk!/lib/armeabi-v7a, /vendor/lib, /system/lib]]
at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:56)
at java.lang.ClassLoader.loadClass(ClassLoader.java:511)
at java.lang.ClassLoader.loadClass(ClassLoader.java:469)
at android.view.LayoutInflater.createView(LayoutInflater.java:583)
at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:764)
... 55 more
Suppressed: java.lang.ClassNotFoundException: android.support.v7.widget.AppCompatImageButtonn
at java.lang.Class.classForName(Native Method)
at java.lang.BootClassLoader.findClass(ClassLoader.java:781)
at java.lang.BootClassLoader.loadClass(ClassLoader.java:841)
at java.lang.ClassLoader.loadClass(ClassLoader.java:504)
... 58 more
Caused by: java.lang.NoClassDefFoundError: Class not found using the boot class loader; no stack trace available
```